apps/client/src/app/implementation-guide-wrapper/r4/implementation-guide.component.ts

```diff
@@ -144,6 +144,11 @@ export class R4ImplementationGuideComponent extends BaseImplementationGuideCompo
     return this.route.snapshot.paramMap.get('implementationGuideId') === 'from-file';
   }
 
+  public fileNameInvalid(page: PageDefinition): boolean {
```

Comment:
minor but...it might be more readable to name this isFileNameValid instead of fileNameInvalid

Reply:
I agree with the naming convention correction but I'm going to keep it with "Invalid" just because it's checking for incorrect characters rather than correct characters.